### ISSUE TITLE
issue with 0.12.0 and 0.13.0

### ISSUE THREAD

**Issue by sh1zen** [keepsimple1/mdns-sd#287]
On windows 11H4, no error has been shown, but the service result unreachable.
same code works until version 0.11.5:

  let mdns = ServiceDaemon::new()?;
    let ip = local_ip()?;
    let host_name = format!("{}.local.", ip);
    let properties = [("port", CAST_SERVICE_PORT)];

    let my_service = ServiceInfo::new(
        LOCAL_DISCOVERY_SERVICE_NAME,
        "ScreenCaster",
        &*host_name,
        ip,
        CAST_SERVICE_PORT,
        &properties[..],
    )?;

    mdns.register(my_service)?;

    Ok(mdns)

**Comment by keepsimple1** [comment]
I tried and indeed that's a bug. The problem is that a new `probing` code in 0.12.0 did not convert service name (including instance name) to lower case properly.  For example, if you change your test instance name to "screencaster", the bug will not show up.

This bug just happened to be fixed by PR #286 .  So if you pull the latest `main` branch, it should be fixed. I tried locally and it seems working. Let me know if the issue still exists. I will try to publish a bugfix release soon.

Thanks for opening this issue!
